src/renderer/services/wallet/types.ts

```diff
@@ -102,13 +102,16 @@ export enum ErrorId {
   SEND_TX
 }
 
+// TODO(@Veado) Move type to clients/type
+
 export type ApiError = {
   errorId: ErrorId
   msg: string
 }
 
 export type NonEmptyApiErrors = NonEmptyArray<ApiError>
 
+// TODO (@thatThorchainGuy) Remove TxRD/TxLD with https://github.com/thorchain/asgardex-electron/issues/573
```

Comment:
hm. dont see a reason here to remove this values in terms of #573. could you clarify it ?

Reply:
Good find! I can't remember why I have put this comment there, it does not make sense. _I will remove it ..._